volttron/platform/vip/proxy_zmq_router.py

```diff
@@ -97,57 +97,57 @@ def startup(self, sender, **kwargs):
         # ----------------------------------------------------------------------------------
         # Create a queue to receive messages from local platform
         # (for example, response for RPC request etc)
-        result = channel.queue_declare(queue=self._rpc_handler_queue,
+        result = channel.queue_declare(self._rpc_handler_queue,
                                        durable=False,
                                        exclusive=True,
                                        auto_delete=True,
                                        callback=None)
-        channel.queue_bind(exchange=connection.exchange,
-                           queue=self._rpc_handler_queue,
+        channel.queue_bind(self._rpc_handler_queue,
+                           connection.exchange,
                            routing_key=self.core.instance_name + '.proxy.router.zmq.outbound.subsystem',
                            callback=None)
-        channel.basic_consume(self.rpc_message_handler,
-                              queue=self._rpc_handler_queue,
-                              no_ack=True)
+        channel.basic_consume(self._rpc_handler_queue,
+                              self.rpc_message_handler
+                              )
```

Comment:
if you remove no_ack=True, do you require a channel.basic_ack(method.delivery_tag) like the one you added in rmq_pubsub.py ?